src/server/game/DataStores/DB2Stores.cpp

```diff
@@ -837,6 +837,20 @@ void DB2Manager::LoadStores(std::string const& dataPath, uint32 defaultLocale)
     for (AzeriteUnlockMappingEntry const* azeriteUnlockMapping : sAzeriteUnlockMappingStore)
         azeriteUnlockMappings[azeriteUnlockMapping->AzeriteUnlockMappingSetID].push_back(azeriteUnlockMapping);
 
+    for (BattlemasterListEntry const* battlemaster : sBattlemasterListStore)
+    {
+        if (battlemaster->MaxLevel < battlemaster->MinLevel)
+        {
+            TC_LOG_ERROR("sql.sql", "Battlemaster (%u) contains bad values for MinLevel (%u) and MaxLevel (%u). Swapping values.", battlemaster->ID, battlemaster->MinLevel, battlemaster->MaxLevel);
```

Comment:
Maybe pick a different logger category for this?

Reply:
updated! Using hotfixes.battlemasterlist now.